src/Illuminate/Contracts/Events/Dispatcher.php

```diff
@@ -26,6 +26,8 @@ public function hasListeners($eventName);
      *
      * @param  object|string  $subscriber
      * @return void
+     *
+     * @throws \Illuminate\Contracts\Container\BindingResolutionException
```

Comment:
Remove this again.

Reply:
so, you mean it is guaranteed at run-time?
because it uses container to resolve the object

Reply:
We don't document cascading throws.

Reply:
So the protected method which calls the container->make() should have the @throw? if I got it right

Reply:
Yeah, looks okay now.